src/main/scala/codacy/plugins/test/PluginsTests.scala

```diff
@@ -14,8 +14,9 @@ object PluginsTests extends ITest {
 
   val opt = "plugin"
 
-  def run(testSources: Seq[Path], dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
-    Printer.green("Running PluginsTests:")
+  def run(testDirectories: Seq[Path], dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/app/Codacy/codacy-plugins-test/pullRequest?prid=4500290)